test/e2e/cluster_samples_operator_test.go

```diff
@@ -80,6 +81,49 @@ func verifyConditionsCompleteSamplesRemoved(sr *samplesapi.SamplesResource) erro
 	})
 }
 
+func verifyClusterOperatorConditionsComplete(t *testing.T) {
+	err := wait.PollImmediate(1*time.Second, 10*time.Minute, func() (bool, error) {
+		state := &configv1.ClusterOperator{
+			TypeMeta: metav1.TypeMeta{
+				APIVersion: configv1.SchemeGroupVersion.String(),
+				Kind:       "ClusterOperator",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				Name: samplesapi.SamplesResourceName,
```

Comment:
looks like this should be named "openshift-cluster-samples-operator" if we want to align w/ the other ones that exist.

Reply:
except for the machine api/config ones but yeah ... will rename